README.md

```diff
@@ -0,0 +1,84 @@
+# relayer
+
+[![Build Status](https://travis-ci.org/wizeline/relayer.svg)](https://travis-ci.org/wizeline/relayer)
+
+
+## Contents
+
+
+1. [About](#about)
+2. [Install](#install)
+3. [Usage](#usage)
+4. [Hacking](#hacking)
+    1. [Setup](#setup)
+    2. [Testing](#testing)
+    3. [Coding conventions](#coding-conventions)
+
+
+## About
+
+Relayer is a library to emit kafka message and group logs to relay them to kafka for log aggregation.
+
+
+## Install
+
+Just do:
```

Comment:
@pablasso will do as soon as have testing just as `sqlalchemy-pagination` is published